### DISCUSSION TITLE
Does shlink have a "private" mode?

### DISCUSSION THREAD

**Discussion by ericswpark** [shlinkio/discussions#134] [Help wanted] [answered]
### Shlink version

N/A

### PHP version

N/A

### How do you serve Shlink

Other (explain in summary)

### Database engine

PostgreSQL

### Database version

N/A

### Summary

Hi, I checked through the documentation but couldn't find anything regarding this. Does shlink support a "private" mode where only I can create short URLs? My idea is to have my own URL shortening instance so I can give out short URLs to others, but I don't want people being able to use shlink on my instance to create short URLs for their own use.

**Comment by acelaya** [comment, marked as answer]
Yes. In fact this is the only way Shlink works.

After setting up your instance, you will have to create an API key. Shlink requires this to be provided in order to create short urls, get visit stats, etc.

Only the short urls themselves are "public", so to speak.
